internal/deviations/deviations.go

```diff
@@ -74,6 +74,11 @@ func P4RTMissingDelete(_ *ondatra.DUTDevice) bool {
 	return *p4rtMissingDelete
 }
 
+// P4rtUnsetElectionIDUnsupported returns whether the device does not support unset election ID.
```

Comment:
please fix the static analysis error. the first word of the comment should be exactly "P4RTUnsetElectionIDUnsupported"

Reply:
And please fix "UseVendorNativeACLConfig" while here